### PR TITLE
[glad] Update to 0.1.33

### PR DESCRIPTION
**Pull request by crackedmind**

### CODE REVIEW

File: ports/glad/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: glad
-Version: 0.1.31
+Version: 0.1.33
```
Comment (NancyLi1013) [inline ports/glad/CONTROL]: Could you please add homepage here?
`Homepage: https://github.com/Dav1dde/glad`

File: ports/glad/portfile.cmake [hunk 2]
```diff
@@ -5,8 +5,8 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/glad/portfile.cmake]: `include(vcpkg_common_functions)` is no longer needed. 
Could you please remove this line?

File: ports/glad/portfile.cmake [hunk 3]
```diff
@@ -5,8 +5,8 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Dav1dde/glad
-    REF 1a42f8a6099c60f6a4522731fadad4191d64e6ff
-    SHA512 02870cf43b5ee33a733122885f748a1368e4487fc08f32ba376d6d53c0efd9ed6e9aea350b723d955869ed47b4d9d69235a52f01723215cf4393d6ca99e2ac00
+    REF de6c39e3040c987323b8ed078c36442f4fb681b3
+    SHA512 a24523186d59de5c0895791c639c62573eaacf1d3843d3bf81eba848b4a33a9a8d17f9b6f791202dac77692bf147e25b3650989731d5ddb7a22e7d023b66885e
     HEAD_REF master
 )
```
Comment (NancyLi1013) [inline ports/glad/portfile.cmake]: Could you update `configure_file(${SOURCE_PATH}/LICENSE ${CURRENT_PACKAGES_DIR}/share/glad/copyright COPYONLY)` as
`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)?`

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Review by vicroms — APPROVED** [review]

**Pull request merged**